crates/fiber-lib/src/rpc/channel.rs

```diff
@@ -293,6 +293,12 @@ pub struct Htlc {
     /// The expiry of the htlc
     #[serde_as(as = "U64Hex")]
     pub expiry: u64,
```

Comment:
I think we do not need to add parameters for these, seems there is no testing scenario for it?